src/Microsoft.OpenApi/Services/OpenApiUrlSpaceNode.cs

```diff
@@ -0,0 +1,231 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.OpenApi.Extensions;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.OpenApi.Services
+{
+    /// <summary>
+    /// A directory structure representing the paths of an OpenAPI document.
+    /// </summary>
+    public class OpenApiUrlSpaceNode
+    {
+        /// <summary>
+        /// All the subdirectories of a node.
+        /// </summary>
+        public IDictionary<string, OpenApiUrlSpaceNode> Children { get; set; } = new Dictionary<string, OpenApiUrlSpaceNode>();
```

Comment:
can be private?  i don't think the setter gets used as well.

Reply:
Resolved